app/controllers/player_sessions_controller.rb

```diff
@@ -0,0 +1,35 @@
+class PlayerSessionsController < ApplicationController
+  skip_before_action(:force_player_sign_in, { :only => [:new_session_form, :add_cookie] })
+
+  def new_session_form
+    render({ :template => "player_sessions/sign_in.html.erb" })
+  end
+
+  def add_cookie
+    player = Player.where({ :email => params.fetch(:email) }).at(0)
+    
+    the_supplied_password = params.fetch(:password)
+    
+    if player != nil
+      they_are_real = player.authenticate(the_supplied_password)
```

Comment:
What do you think about renaming this variable to `are_they_real`?

Reply:
I do like having the variable name be more of a question. I think it makes that particular line slightly more readable, but I think I like having the conditional for

```ruby
if they_are_real == false
```

better. I think this reads very well and changing it to `if are_they_legit == false` loses some of that readability.

Reply:
I agree that the conditional reads better with `they_are_real`, but the conditional comes later. By then, if they already understand what is going on, it's okay to sacrifice some readability, I think.

The declarative-ness of `they_are_real` when they first get there is a little confusing, I think, when they aren't very solid yet on the idea of `authenticate` and the fact that it returns `true` or `false`. I'm leaning towards making their first contact with the idea more easily understood with `are_they_real`.